### DISCUSSION TITLE
Path generation With script

### DISCUSSION THREAD

**Discussion by squadmind1** [ImageMagick/ImageMagick#1957] [General]
I have a rather specific and esoteric problem, or at least I think so.

Ultimately, I want to generate a path from a set of points/vectors/contours via a script. This path should then be copied onto a jpg (the original source of the points/curves/contours). The resulting image has to have the path we created visible and editable in Photoshop.

All of this happens inside a Linux server; I work in Java mostly. To be clear, this has to work via CLI.

My questions are:
1) Is this possible at all?
2) What do the parameters need to look like if it is?

What I found in my quest to solve this problem is the 8BIM path specification from adobe, but that's about all I have right now. Maybe there even is a way to directly write curves programmatically into the jpg metadata?

I would be very grateful if someone can give me some pointers in the right direction.

**Comment by fmw42** [comment]
ImageMagick cannot currently write Photoshop paths into an image as far as I know. I do not think you can even write the equivalent SVG path into the meta data, though you could create that by script you points external to ImageMagick. Perhaps you can import that with your image into Photoshop.
